content/en/_index.html

```diff
@@ -8,52 +8,61 @@
 
 {{< blocks/section id="oceanNodes" >}}
 {{% blocks/feature image="flower" %}}
-[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
 
-It groups containers that make up an application into logical units for easy management and discovery. Kubernetes builds upon [15 years of experience of running production workloads at Google](http://queue.acm.org/detail.cfm?id=2898444), combined with best-of-breed ideas and practices from the community.
+[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open-source system for automating
+deployment, scaling, and management of containerized applications.
+
+It groups containers that make up an application into logical units for easy management and discovery. Kubernetes builds
+upon [15 years of experience of running production workloads at Google](http://queue.acm.org/detail.cfm?id=2898444),
+combined with best-of-breed ideas and practices from the community.
+
 {{% /blocks/feature %}}
 
 {{% blocks/feature image="scalable" %}}
 #### Planet Scale
 
-Designed on the same principles that allow Google to run billions of containers a week, Kubernetes can scale without increasing your operations team.
+Designed on the same principles that allow Google to run billions of containers a week, Kubernetes can scale without
+increasing your operations team.
 
 {{% /blocks/feature %}}
 
 {{% blocks/feature image="blocks" %}}
 #### Never Outgrow
 
-Whether testing locally or running a global enterprise, Kubernetes flexibility grows with you to deliver your applications consistently and easily no matter how complex your need is.
+Whether testing locally or running a global enterprise, Kubernetes flexibility grows with you to deliver your
+applications consistently and easily no matter how complex your need is.
 
 {{% /blocks/feature %}}
 
 {{% blocks/feature image="suitcase" %}}
 #### Run K8s Anywhere
 
-Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.
+Kubernetes is open source,giving you the freedom to take advantage of on-premises, hybrid, or public cloud
```

Comment:
Non-blocking nit.
```suggestion
Kubernetes is open source, giving you the freedom to take advantage of on-premises, hybrid, or public cloud
```

Reply:
This PR is not meant to be merged. It is not meant to change these files at all.
A trivial change is incorporated to make sure that there is something new to trigger a build.